docs/conf.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'ppb-vector'
+copyright = '2019, Piper Thunstrom, Jamie Bliss'
+author = 'Piper Thunstrom, Jamie Bliss'
+
+# The short X.Y version
+version = ''
```

Comment:
Is this autofilled?

Reply:
Not as far as I know? I'm also not sure where it's used.